sklearn/preprocessing/data.py

```diff
@@ -397,6 +397,261 @@ def inverse_transform(self, X, copy=None):
         return X
 
 
+class RobustScaler(BaseEstimator, TransformerMixin):
+    """Standardize features by removing the median and scaling to IQR.
+
+    Centering and scaling happen independently on each feature (or each
+    sample, depending on the `axis` argument) by computing the relevant
+    statistics on the samples in the training set. Median and  interquartile
+    range are then stored to be used on later data using the `transform`
+    method.
+
+    Standardization of a dataset is a common requirement for many
+    machine learning estimators. Typically this is done by removing the mean
+    and scaling to unit variance. However, outliers can often influence the
+    sample mean / variance in a negative way. In such cases, the median and
+    the interquartile range often give better results.
+
+    Parameters
+    ----------
+    interquartile_scale: float or string in  ["normal" (default), ],
+           The interquartile range is divided by this factor. If
```

Comment:
I am not sure I understand the parametrization here. So we are always using 1/4 3/4 quantiles and then scale them? Is that a standard practice? [sorry I rarely do robust statistics]